<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="View" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuBrush">
    <property name="title">
     <string>&amp;Brush</string>
    </property>
    <widget class="QMenu" name="menuWidth">
     <property name="title">
      <string>&amp;Width</string>
     </property>
     <addaction name="action1px"/>
     <addaction name="action2px"/>
     <addaction name="action5px"/>
     <addaction name="action10px"/>
     <addaction name="actionMore_Sizes"/>
    </widget>
    <widget class="QMenu" name="menuColor">
     <property name="title">
      <string>&amp;Color</string>
     </property>
     <addaction name="actionBlack"/>
     <addaction name="actionWhite"/>
     <addaction name="actionRed"/>
     <addaction name="actionBlue"/>
     <addaction name="actionGreen"/>
     <addaction name="actionRainbow"/>
     <addaction name="actionMore_Colors"/>
    </widget>
    <addaction name="menuWidth"/>
    <addaction name="menuColor"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuBrush"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="actionSave_As"/>
   <addaction name="separator"/>
   <addaction name="action1px"/>
   <addaction name="action2px"/>
   <addaction name="action5px"/>
   <addaction name="action10px"/>
   <addaction name="separator"/>
   <addaction name="actionWhite"/>
   <addaction name="actionRed"/>
   <addaction name="actionBlue"/>
   <addaction name="actionGreen"/>
   <addaction name="actionBlack"/>
  </widget>
  <action name="actionNew">
   <property name="text">
    <string>&amp;New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>&amp;Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>&amp;Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>Save &amp;As</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>&amp;Exit</string>
   </property>
  </action>
  <action name="action1px">
   <property name="text">
    <string>&amp;1px</string>
   </property>
  </action>
  <action name="action2px">
   <property name="text">
    <string>&amp;2px</string>
   </property>
  </action>
  <action name="action5px">
   <property name="text">
    <string>&amp;5px</string>
   </property>
  </action>
  <action name="action10px">
   <property name="text">
    <string>&amp;10px</string>
   </property>
  </action>
  <action name="actionBlack">
   <property name="text">
    <string>&amp;Black</string>
   </property>
  </action>
  <action name="actionWhite">
   <property name="text">
    <string>&amp;White</string>
   </property>
  </action>
  <action name="actionRed">
   <property name="text">
    <string>&amp;Red</string>
   </property>
  </action>
  <action name="actionBlue">
   <property name="text">
    <string>&amp;Blue</string>
   </property>
  </action>
  <action name="actionGreen">
   <property name="text">
    <string>&amp;Green</string>
   </property>
  </action>
  <action name="actionMore_Colors">
   <property name="text">
    <string>More Colors...</string>
   </property>
  </action>
  <action name="actionMore_Sizes">
   <property name="text">
    <string>More Sizes...</string>
   </property>
  </action>
  <action name="actionRainbow">
   <property name="text">
    <string>Rainbow</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>View</class>
   <extends>QWidget</extends>
   <header>view.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
